src/components/application_manager/test/commands/mobile/put_file_test.cc

```diff
@@ -344,6 +345,53 @@ TEST_F(PutFileRequestTest, Run_InvalidPutFile_UNSUCCESS) {
   command->Run();
 }
 
+TEST_F(PutFileRequestTest, Run_SrcSumEqual_SendSuccessResponse) {
+  (*msg_)[am::strings::msg_params][am::strings::crc32_check_sum] = 2768625435u;
+
+  ExpectReceiveMessageFromSDK();
+  ON_CALL(app_mngr_, get_settings())
+      .WillByDefault(ReturnRef(app_mngr_settings_));
+  ON_CALL(app_mngr_settings_, app_storage_folder())
+      .WillByDefault(ReturnRef(kStorageFolder));
+  ON_CALL(*mock_app_, folder_name()).WillByDefault(Return(kFolder));
+  const uint32_t avaliable_space = binary_data_.size() + 1;
+  ON_CALL(*mock_app_, GetAvailableDiskSpace())
+      .WillByDefault(Return(avaliable_space));
+  ON_CALL(*mock_app_, AddFile(_)).WillByDefault(Return(true));
+
+  const std::string file_path = kStorageFolder + "/" + kFolder;
+  EXPECT_CALL(app_mngr_, SaveBinary(binary_data_, file_path, kFileName, 0u))
+      .WillOnce(Return(mobile_apis::Result::SUCCESS));
+  EXPECT_CALL(*mock_app_, increment_put_file_in_none_count());
+  ExpectManageMobileCommandWithResultCode(mobile_apis::Result::SUCCESS);
+  PutFileRequestPtr command(CreateCommand<PutFileRequest>(msg_));
+  ASSERT_TRUE(command->Init());
+  command->Run();
+  EXPECT_TRUE(file_system::RemoveDirectory(kStorageFolder, true));
```

Comment:
@BSolonenko Do you create something on file system? Why do you make deletion?

Reply:
@AByzhynar Fixed in https://github.com/smartdevicelink/sdl_core/pull/2139/commits/17cae31dfeb5a8e55281633ebe3efc6dd051c614